whylog/converters/__init__.py

```diff
@@ -19,6 +21,15 @@ class AbstractConverter(object):
     def convert(cls, pattern_group):
         raise NotImplementedError
 
+    @classmethod
+    def safe_convert(self, pattern_group):
```

Comment:
Is there any reason to write two functions `safe_converter` and `converter`? Will LogReader not safe converter?